<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>621</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="vd" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="vddio" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="v18" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="biasd" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="biasr" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ninput">
     <attribute name="title">
      <string>N Input</string>
     </attribute>
     <widget class="CustomSlider" name="vana_n" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vcascn" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vn0" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vn1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vn2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vprfb" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pinput">
     <attribute name="title">
      <string>P Input</string>
     </attribute>
     <widget class="CustomSlider" name="vana_p" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vcascp" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vp0" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vp1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="vp2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="readout">
     <attribute name="title">
      <string>Readout</string>
     </attribute>
     <widget class="CustomSlider" name="biasro" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="biasio" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomSlider" name="offset" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="CustomSlider" name="vfb" native="true">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>472</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="vcal" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomSlider" name="vhold" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>620</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="unzoom_y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>41</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string notr="true">U
n
z
o
o
m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unzoom_x">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Unzoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="darkfield">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>785</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CustomSlider" name="average" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>740</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>790</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark field correction</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>690</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="image">
         <property name="text">
          <string>Single</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loop">
         <property name="text">
          <string>Loop</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delayScan">
         <property name="text">
          <string>Delay scan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="takeData">
         <property name="text">
          <string>Take Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Events</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numberOfEvents">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Signal source:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="internalCal">
           <property name="text">
            <string>internal cal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="laser">
           <property name="text">
            <string>laser</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="externalCal">
           <property name="text">
            <string>source</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Readout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="columnwise">
         <property name="text">
          <string>column wise</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rowwise">
         <property name="text">
          <string>row wise</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QWidget</extends>
   <header>customslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
